### ISSUE TITLE
Wrong comment type for html file

### ISSUE THREAD

**Issue by psnszsn** [tyru/caw.vim#109]
I have `context_filetype.vim` installed and the html file looks like this:
```html
<script>
    let a = 1;
</script>

<div>
    test
</div>

<style>
    div{
        border-top: 1px;
    }
</style>
```

Commenting works fine in the style tag and in the div tag, but once I comment the line in the script tag, commenting the div won't use the proper style of comment anymore. I expect `<!-- -->` html comments but I get  `//` javascript comments.

I don't think this is a `context_filetype.vim` issue, since `:echo context_filetype#get_filetype()` returns `html` as expected.

**Comment by tyru** [comment]
@psnszsn could you try #111 ?

**Comment by tyru** [comment]
ah, #111 doesn't solve this.

caw.vim changes `let &filetype = context_filetype#get_filetype()`
if `&filetype != context_filetype#get_filetype()`.
It causes ftplugin load like MaxMEllon/vim-jsx-pretty, and so on.

caw.vim behavior is bad because it may cause unwanted side-effect like this issue's problem.

---

Could you try `devel` branch?
It should fix this issue.

**Comment by tyru** [comment]
Fixed in #118
